docs/bridge/developer_hub.mdx

```diff
@@ -4,26 +4,67 @@ sidebar_position: 4
 sidebar_label: Developer Hub
 ---
 
-import { eth, vara } from "@site/src/addresses/bridge-addresses.js";
+import { eth, ethMainnet, vara, varaMainnet } from "@site/src/addresses/bridge-addresses.js";
 
 # Developer Hub
 
 This page provides technical resources for developers and integrators working with the Vara ↔ Ethereum Bridge.  
-It includes the addresses of deployed smart contracts and on-chain programs on both Ethereum (**Hoodi testnet**) and Vara Testnet.
+It includes the addresses of deployed smart contracts and on-chain programs on both Ethereum and Vara networks.
```

Comment:
we have to maintain info about both - mainet and testnet (for integration, development, testing purposes - for the ecosystem projects before they launch on the mainnet)!